README.md

```diff
@@ -89,14 +89,19 @@ The `:credential` can also reference a keystore using a map with the following p
 
 ### Responses
 
-Basic usage for responses from the IdP looks like:
+Basic usage for responses from the IdP looks like this (assuming a Ring `request`):
 
 ```clj
 (require '[saml20-clj.core :as saml])
-
-;; Coerce the response to an OpenSAML `Response`. This can be anything from a raw XML string to a parsed
-;; `org.w3c.dom.Document`
-(-> (saml/->Response xml)
+(require '[saml20-clj.encode-decode :as saml-decode])
+
+(-> request
+    :params
+    :SAML-response
+    (saml-decode/base64->str)
```

Comment:
you don't need to wrap lone function name symbols in parens in the threading macros if you're not passing addition args  e.g.

```clj
(-> x
    y)
```

is the same as 

```clj
(-> x
    (y))
```

It's more idiomatic not to wrap symbols in threading macros unless it's needed. Mind tweaking the examples?

Reply:
b60e651